$NetBSD$

QNX seems to lack the definition of OPEN_MAX, too

--- src/fc/fslibos.h.orig	2007-09-05 02:18:23.000000000 +0200
+++ src/fc/fslibos.h
@@ -64,7 +64,7 @@ from The Open Group.
 # endif
 #endif
 #ifndef OPEN_MAX
-#if defined(SVR4) || defined(__UNIXOS2__)
+#if defined(SVR4) || defined(__UNIXOS2__) || defined(__QNX__)
 #define OPEN_MAX 256
 #else
 #include <sys/param.h>
